src/rpc/blockchain.cpp

```diff
@@ -323,6 +324,21 @@ UniValue waitforblockheight(const JSONRPCRequest& request)
     return ret;
 }
 
+UniValue syncwithvalidationinterfacequeue(const JSONRPCRequest& request)
```

Comment:
Not sure but `src/rpc/misc.cpp` is a better place?

Reply:
My understanding is that `blockchain.cpp` also contains mempool rpcs. And this one is about mempool, no?

Reply:
Ok.